costing.md

```diff
@@ -1,37 +1,44 @@
 # Project Costs Breakdown
 
 ## Website
-- **Website domain**: 1000
-- **Website backend hosting**: 600
+- **Website domain**: 1000 : https://www.godaddy.com/en-in/domainsearch/find?domainToCheck=iris-mitwpu.edu
+- **Website backend hosting**: 600 : https://railway.app/pricing
 
 ## Essential Hardware Components
-- **Linear Actuator**: 3300
-- **STM32 board**: 1400
-- **USB to OBD**: 250
+- **Linear Actuator**: 3300 : https://robu.in/product/linear-actuator-stroke-length-100mm7mm-s1500n12v/
+- **STM32 board**: 1400 : https://robu.in/product/stmicroelectronics-development-board-stm32-nucleo-64-mcu-arduino-uno-v3-connectivity-flexible-power-supply/
+- **USB to OBD**: 250 : https://robu.in/product/mini-v2-1-elm327-obd2-bluetooth-interface-auto-car-scanner/
```

Comment:
This is wireless , we require a wired board config wire .
try adding that , local source can be done .